### PR TITLE
etcdhttp: use leveled logging

### PR DESCRIPTION
**Pull request by xiang90**
make etcdhttp use leveled logging.

the next step is to add debugging level log for incoming http requests.

### CODE REVIEW

File: etcdserver/etcdhttp/client.go [hunk 1]
```diff
@@ -585,7 +584,7 @@ func handleKeyWatch(ctx context.Context, w http.ResponseWriter, wa store.Watcher
 			ev = trimEventPrefix(ev, etcdserver.StoreKeysPrefix)
 			if err := json.NewEncoder(w).Encode(ev); err != nil {
 				// Should never be reached
-				log.Printf("error writing event: %v\n", err)
+				plog.Warningf("error writing event: %v\n", err)
```
Comment (yichengq) [inline etcdserver/etcdhttp/client.go]: (%v)

File: etcdserver/etcdhttp/client_security.go [hunk 2]
```diff
@@ -115,7 +114,7 @@ func hasGuestAccess(sec *security.Store, r *http.Request, key string) bool {
 	if role.HasKeyAccess(key, writeAccess) {
 		return true
 	}
-	log.Printf("security: Invalid access for unauthenticated user on resource %s.", key)
+	plog.Warningf("security: invalid access for unauthenticated user on resource %s.", key)
```
Comment (yichengq) [inline etcdserver/etcdhttp/client_security.go]: remove the `security: ` prefix? /cc @barakmich

File: etcdserver/etcdhttp/client_security.go [hunk 3]
```diff
@@ -157,7 +156,7 @@ func (sh *securityHandler) baseRoles(w http.ResponseWriter, r *http.Request) {
 	rolesCollections.Roles = roles
 	err = json.NewEncoder(w).Encode(rolesCollections)
 	if err != nil {
-		log.Println("etcdhttp: baseRoles error encoding on", r.URL)
+		plog.Warningf("baseRoles error encoding on", r.URL)
```
Comment (yichengq) [inline etcdserver/etcdhttp/client_security.go]: Warningln

Comment (yichengq) [inline etcdserver/etcdhttp/client_security.go]: or "baseRoles error encoding on %s (%v)", r.URL, err

File: etcdserver/etcdhttp/http.go [hunk 4]
```diff
@@ -38,7 +38,10 @@ const (
 	defaultWatchTimeout = time.Duration(math.MaxInt64)
 )
 
-var errClosed = errors.New("etcdhttp: client closed connection")
+var (
+	plog      = capnslog.NewPackageLogger("github.com/coreos/etcd", "etcdhttp")
```
Comment (yichengq) [inline etcdserver/etcdhttp/http.go]: `etcdserver/etcdhttp` ?

Comment (xiang90) [inline etcdserver/etcdhttp/http.go]: this is the pkg name. it should be just `etcdhttp`. it is short and clear in my opinion.

File: etcdserver/etcdhttp/client_security.go (hunk 2 above)
Comment (xiang90) [inline etcdserver/etcdhttp/client_security.go]: we should keep the security prefix i hope.

**Comment by xiang90** [comment]
@yichengq @barakmich I am going to merge this since it is low risk.

**Pull request merged**